src/components/templates/product-table/use-product-column.tsx

```diff
@@ -21,6 +22,34 @@ const useProductTableColumn = ({ setTileView, setListView, showList }) => {
     }
   }
 
+  const getProductSalesChannels = (salesChannels) => {
+    console.log(salesChannels)
+    if (salesChannels?.length) {
+      return (
+        <span className="inter-small-regular">
+          {salesChannels[0].name}
+          {salesChannels.length > 1 && (
+            <Tooltip
+              content={
+                <div className="flex flex-col">
+                  {salesChannels.slice(1).map((sc) => (
```

Comment:
discussion: we could add a sort for sales channels that is reused across SC list components e.g. sort the default channel first and rest by name, wdyt?

Reply:
I added a HoF that returns the compare operator you've used when given the default sales channel id 😄 I thought it would be the most elegant way of doing it